doc/source/commands/communication.rst

```diff
@@ -143,3 +143,68 @@ The receiving CPU will use :attr:`CORE:MESSAGES` to access its message queue::
   } ELSE {
     PRINT "Unexpected message: " + RECEIVED:CONTENT.
   }
+
+.. _connectivityManagers:
+
+Connectivity Managers
+---------------------
+
+.. note::
+
+    .. versionadded:: v1.0.2
+        The concept of selectable connectivity managers was added after KSP
+        introduced a stock communications system (CommNet). kOS was updated to
+        support both CommNet and RemoteTech. Other mods may be supported or
+        provide their own support in the future.
+
+kOS can implement communications over a variaty of connectivity configurations.
+We refer to these options as "Connectivity Managers." You can slect the active
+manager from the :ref:`kOS section of KSP's Difficulty Settings<settingsWindow>`.
+If the currently selected manager no longer is available, or if a new manager
+becomes available, you will be prompted with a dialog box to select the manager
+you want to use.
+
+.. object:: Connectivity Managers
+
+    By default kOS supports the following list of managers. If a manager is not
+    currently available (because the required mod isn't installed, or the system
+    is disabled in the settings) it will not be shown in lists of available managers.
+
+    StockConnectivityManager
```

Comment:
Again, same comment as in the code review about using the word "stock" to mean "not CommNet", when "CommNet" is a stock feature - its a bit confusing.